toolkit/partner-chains-cli/src/create_chain_spec/mod.rs

```diff
@@ -223,6 +235,24 @@ impl CreateChainSpecConfig {
 	}
 }
 
+impl<T> Default for CreateChainSpecConfig<T> {
+	fn default() -> Self {
```

Comment:
I bet there is a macro for it, but IDK if I want to look for it and add extra crate for it.